scripts/freeze.mjs

```diff
@@ -17,6 +19,17 @@ import {
 
 const [version, releaseDate] = process.argv.slice(2);
 
+const argv = yargs(hideBin(process.argv))
+  .boolean('commit')
+  .default('commit', false)
+  .describe('commit', '`true` if the state of the repo should be committed automatically at the end of the freeze' +
+    ' process.')
+  .boolean('push')
+  .default('push', false)
+  .describe('push', '`true` if the state of the repo should be pushed to remote automatically at the end of the' +
+    ' freeze process.')
+  .argv;
+
```

Comment:
```suggestion
const argv = yargs(hideBin(process.argv))
  .boolean('commit')
  .default('commit', false)
  .describe('commit', '`true` if the state of the repo should be committed automatically at the end of the freeze' +
    ' process.')
  .boolean('push')
  .default('push', false)
  .describe('push', '`true` if the state of the repo should be pushed to remote automatically at the end of the' +
    ' freeze process.')
  .argv;

const [version, releaseDate] = argv._;
```
Let's the `yargs` manage all CLI arguments. Under the `_` key there are non-hyphened options (https://github.com/yargs/yargs/blob/master/docs/examples.md#and-non-hyphenated-options-too-just-use-argv_)

Reply:
Ah, I was looking for it but didn't find it 👌